applications/atracsys_visualizer/cpp/atracsys_visualizer.cpp

```
@@ -0,0 +1,282 @@
+    add_flow(mode_switcher, holoviz, {{"out_specs", "input_specs"}});
+  }
+
+ private:
+  std::string config_path_;
+  std::string source_{"replayer"};
+  std::string data_path_{"data/atracsys_visualizer"};
+};
+
+int main(int argc, char** argv) {
+  CommandLineOptions options;
+  if (!parse_arguments(argc, argv, options)) {
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function that is an entry point.

(throwInEntryPoint)